core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -423,6 +438,12 @@ public static ConfigDef baseSchemaRegistryConfigDef() {
     .define(SCHEMA_COMPATIBILITY_CONFIG, ConfigDef.Type.STRING, COMPATIBILITY_DEFAULT,
         ConfigDef.Importance.HIGH, COMPATIBILITY_DOC
     )
+    .define(SCHEMA_CACHE_SIZE_CONFIG, ConfigDef.Type.INT, SCHEMA_CACHE_SIZE_DEFAULT,
+        ConfigDef.Importance.LOW, SCHEMA_CACHE_SIZE_DOC
+    )
+    .define(SCHEMA_CACHE_EXPIRY_SECS_CONFIG, ConfigDef.Type.INT, SCHEMA_CACHE_EXPIRY_SECS_DEFAULT,
+        ConfigDef.Importance.LOW, SCHEMA_CACHE_EXPIRY_SECS_DOC
```

Comment:
So this is setting up a LRU cache with 5 mins expiry?

Reply:
Yes